### ISSUE TITLE
pass slices z dimension of 3d volume as batch dimension

### ISSUE THREAD

**Issue by bartmch** [Project-MONAI/MONAI#808]
Hi, I have a volume with 3 color channels and 50 slices (e.g. c,x,y,z => 3,512,512,50) and I would like to feed this into a 2D classification model. I understand that for 2D classification the slices (z) can be seen as the batch size. How can I implement this into Monai? Are there any transforms that can do this operation?

**Comment by wyli** [comment]
could consider
- loading image into [C, H, W, D] shape
- ``RandSpatialCropd(roi_size=[-1,-1, 1])`` to randomly sample slices during training (output shape [C, H, W, 1])
- ``SqueezeDim(dim=3)`` (output shape [C, H, W])
the dataloader will add a new batch dim, making the output shape [B, C, H, W]
